2021/Robot2021/src/main/cpp/subsystems/Drivetrain.cpp

```diff
@@ -860,6 +860,7 @@ void Drivetrain::RamseteFollowerExecute(void)
 
 bool Drivetrain::RamseteFollowerIsFinished(void)
 {
+    m_trajTimer.Stop();

```

Comment:
Did you intend to add this line in this change?

Reply:
I was trying to make this a separate pull request but I think I messed up somewhere along the process

Reply:
By calling the Stop() method, the timer will actually stop. But when is RamseteFollowerIsFinished() called? Is it only called when the command actually ends?